src/middleware/errorHandler.js

```diff
@@ -125,6 +127,15 @@ const filterQuery = (url) => {
 	return newUrl;
 };
 
+const handleSilentError = (error, req, res, next) => {
+	if (error.catchedError instanceof SilentError && SILENT_ERROR_ENABLED) {
```

Comment:
I would like to always ensure the message is written as code 200 and success:'success', and only add the x-silent-error when the env variable is set.

That way, no matter the deployment, abuse of the route is prevented, and if the proper setup is given the infrastructure is also informed of the error (but the header should only be set if the infrastructure is setup to remove it).